pkg/cli/obots_rm.go

```diff
@@ -0,0 +1,92 @@
+package cli
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/obot-platform/obot/apiclient/types"
+	"github.com/obot-platform/obot/pkg/system"
+	"github.com/spf13/cobra"
+)
+
+// DeleteObot implements the 'obot obot rm' subcommand
+type DeleteObot struct {
+	root  *Obot
+	Force bool `usage:"Skip confirmation prompt" short:"f"`
+}
+
+func (c *DeleteObot) Customize(cmd *cobra.Command) {
+	cmd.Use = "rm [ID...]"
+	cmd.Short = "Delete one or more obots"
+	cmd.Long = "Delete one or more obots by ID"
+	cmd.Aliases = []string{"remove", "delete"}
+}
+
+func (c *DeleteObot) Run(cmd *cobra.Command, args []string) error {
+	if len(args) == 0 {
+		return errors.New("at least one obot ID must be provided")
+	}
+
+	// Collect valid IDs first and validate them
+	var validIDs []string
+	var validProjects []*types.Project
+	var errs []error
+
+	for _, id := range args {
+		// Check if ID has the project prefix (both p1- format and p1* format)
+		if !strings.HasPrefix(id, system.ProjectPrefix+"-") && !strings.HasPrefix(id, system.ProjectPrefix) {
```

Comment:
This catches both, so I think this is fine.
```suggestion
		!strings.HasPrefix(id, system.ProjectPrefix) {
```